src/helpers/templates/get_guild_templates.ts

```diff
@@ -16,5 +17,10 @@ export async function getGuildTemplates(guildId: string) {
     endpoints.GUILD_TEMPLATES(guildId),
   )) as DiscordTemplate[];
 
-  return templates.map((template) => structures.createTemplateStruct(template));
```

Comment:
Aren't we going to remove templateStruct? Thats why I did not rename it to DiscordenoGuildTemplate in this case.